pkg/apis/migration/migration.go

```diff
@@ -263,17 +259,16 @@ func (volumeMigration *volumeMigration) GetVolumePath(ctx context.Context, volum
 	var fileBackingInfo *vim25types.BaseConfigInfoDiskFileBackingInfo
 	bUseVslmAPIs, err := common.UseVslmAPIs(ctx, vCenter.Client.ServiceContent.About)
 	if err != nil {
-		msg := fmt.Sprintf("Error while determining the correct APIs to use for vSphere version %q, Error= %+v", vCenter.Client.ServiceContent.About.ApiVersion, err)
-		log.Errorf(msg)
-		return "", errors.New(msg)
+		return "", logger.LogNewErrorf(log,
+			"Error while determining the correct APIs to use for vSphere version %q, Error= %+v",
```

Comment:
Error -> error

Same comment applies to other messages.

Reply:
@xing-yang Do you mind if I address this preexisting issue in separate reviews? I prefer to fix one anti pattern at a time, which is easier for reviewers (and me to mechanically change the code).

I do fix multiple anti-patterns together, if the change is small. But, I feel we need to get closure on one anti-pattern first, before moving to another.

Of course, if you insist, I will sweep through the file in the same change.

Reply:
Sure.